<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackJeckTableWidget</class>
 <widget class="QWidget" name="BlackJeckTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#BlackJeckTableWidget{
	background-color: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDeck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid  rgb(255, 255, 255);
	border-radius: 10px;
	border-image: url(:/Games/BlackDjack/assets/Колода.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelCard_1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid  rgb(255, 255, 255);
	border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCard_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid  rgb(255, 255, 255);
	border-radius: 10px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCard_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid  rgb(255, 255, 255);
	border-radius: 10px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCard_4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid  rgb(255, 255, 255);
	border-radius: 10px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCard_5">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid  rgb(255, 255, 255);
	border-radius: 10px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCard_6">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid  rgb(255, 255, 255);
	border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
